browser/src/view/components/Button/Button.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  return (
+    <button
+      className={`${
+        type === ButtonAttributes.Type.destructive
+          ? 'btn-destructive'
+          : type === ButtonAttributes.Type.primary
+          ? 'btn-primary'
+          : 'btn-secondary'
+      }
+      ${
+        type === ButtonAttributes.Layout.iconAndLabel
+          ? 'btn-icon-label'
+          : type === ButtonAttributes.Layout.iconOnly
+          ? 'btn-icon-only'
+          : 'btn-label-only'
+      }`}
+    >
+      {layout === ButtonAttributes.Layout.iconOnly ||
+      layout === ButtonAttributes.Layout.iconAndLabel
+        ? Icon
+        : null}
+      {layout === ButtonAttributes.Layout.labelOnly ||
+      layout === ButtonAttributes.Layout.iconAndLabel
+        ? label
+        : null}
+    </button>
```

Comment:
For button labels without button text: https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html